elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -2,7 +2,7 @@ defmodule API.Client.Channel do
   use API, :channel
   alias API.Client.Views
   alias Domain.Instrumentation
-  alias Domain.{Clients, Resources, Gateways, Relays}
+  alias Domain.{Clients, Resources, Gateways, Relays, Flows}
```

Comment:
Hm, maybe this is a nit, but do you think we might call these `authorizations` instead? Maybe I spent too much time at Cisco but I think of layer 3 netflow-style logs (flow size, ip/port tuples) when I think Flows. I think these will probably need to be reported by connlib (see #2060 )

Reply:
@jamilbk Flow has the source/dest IP, resource which has traffic filter rules, and the plan is to add metrics that will allow flow to show the data volume/activity spots. Authorizations will be ambiguous because we already have Auth context and it's a heavily overloaded term :/.